Pages/Page 1_Intro TOC.md

```diff
@@ -72,5 +90,21 @@ work the same.
     heroku-toolbelt/3.25.0 (x86_64-linux-gnu) ruby/2.1.5
     You have no installed plugins.
 
-With the tools installed, we can begin working on our web app.
+Only enter the first line in the command line; the following lines are the results of your command and indicate the versions of software you have installed on your computer.
+
+If your commands and results look like the example above, you are ready to move on and start working on your Chatter app.
+
+
+
+## Moving Forward
+Once you confirm your server is working properly, move to the next page where were will briefly go over what Clojure is, what the Web is and how we interact with it, and how HTML works.
+
+Next page, [Chapter 2: Big Picture]
+[Chapter 2: Big Picture]: Page_2_Big_Picture
+
+
+### Copyright and License
+
+Copyright © 2015 Chris Koehnen
```

Comment:
Completely off topic, we should probably look at assigning the copyrights to bridgefoundry or clojurebridge nat.